libraries/AP_HAL/SPIDevice.h

```diff
@@ -66,6 +66,12 @@ class SPIDeviceManager {
     {
         return nullptr;
     }
+
+    /* Return spi count number table. */
+    virtual uint8_t get_count() { return 0; }
```

Comment:
Ok, i understand, but i think it's will be necessary to capture in the top of HAL until all device interfaces would be compatible with the new interface.